content/techniques/caching.md

```diff
@@ -10,6 +10,13 @@ First install [required packages](https://github.com/node-cache-manager/node-cac
 $ npm install cache-manager
 ```
 
+> warning **Warning**  As of version `>=9.2.1` of NestJS is compatible with both `cache-manager` v4 and v5.
+
+> warning **Warning**  `Cache-manager` version 4 uses seconds for `TTL (Time-To-Live)`. Current version of `cache-manager` (v5) has switched to using milliseconds instead. NestJS doesn't convert the value, and simply forwards the ttl you provide to the library. In other words:
+> * If using `cache-manager` v4, provide ttl in seconds
+> * If using `cache-manager` v5 or newer, provide ttl in milliseconds
```

Comment:
Let's not say "or newer" as we don't know what v6 will be like

Reply:
so could it go?
```suggestion
> * If using `cache-manager` v5, provide ttl in milliseconds
```

Reply:
Exactly

Reply:
Done! ✅ change made 🙌